worker/upgradesteps/worker.go

```diff
@@ -40,6 +41,14 @@ var (
 	UpgradeStartTimeoutController = time.Minute * 15
 )
 
+// DefaultRetryStrategy is the retry strategy for performing upgrades that should
+// be used to construct a NewWorker under most circumstances.
+var DefaultRetryStrategy = retry.CallArgs{
```

Comment:
Instead of having an exported variable, let's just declare this inside the manifold start func.